### PR TITLE
Apply black to test_Prank_tool.py

### PR DESCRIPTION
**Pull request by jvfe**
This pull request addresses issue #2552 

- [x] I hereby agree to dual licence this and any previous contributions under both
the _Biopython License Agreement_ **AND** the _BSD 3-Clause License_.

- [x] I have read the ``CONTRIBUTING.rst`` file, have run ``flake8`` locally, and
understand that AppVeyor and TravisCI will be used to confirm the Biopython unit
tests and style checks pass with these changes.

- [x] I have added my name to the alphabetical contributors listings in the files
``NEWS.rst`` and ``CONTRIB.rst`` as part of this pull request, am listed
already, or do not wish to be listed. (*This acknowledgement is optional.*)

### CODE REVIEW

File: Tests/test_Prank_tool.py [hunk 1]
```diff
@@ -26,9 +26,11 @@
     # For Windows, PRANK just comes as a zip file which contains the
     # prank.exe file which the user could put anywhere.  We'll try a few
     # sensible locations under Program Files... and then the full path.
-    likely_dirs = ["",  # Current dir
-                   prog_files,
-                   os.path.join(prog_files, "Prank")] + sys.path
+    likely_dirs = [
+        "",  # Current dir
```
Comment (peterjc) [inline Tests/test_Prank_tool.py:30]: That looks like an old bug, "" does not mean current directory but "." would. If you have access to Windows would you be able to try this?

**Comment by peterjc** [comment]
This looks good, thank you. We'll let the automated checks finish before merging though.

File: Tests/test_Prank_tool.py (hunk 1 above)
Comment (jvfe) [reply Tests/test_Prank_tool.py:30]: Oh, didn't catch that. Don't have access to windows though, unfortunately.

Comment (peterjc) [reply Tests/test_Prank_tool.py:30]: Nor me anymore, unfortunately. Perhaps it really does work on Windows? Let's leave it alone for now.

**Comment by peterjc** [comment]
Merged, thank you. Let us know if you want to be named in the CONTRIB file or NEWS entry - or include this in your next pull request?

**Pull request merged**